File: hw/vizcaino/u3/hw26TemplateMethod/hw26TemplateMethodPython/beverage_test.py
from tea import Tea
from coffee import Coffee

class BeverageTest:
    @staticmethod
    def main():
        tea = Tea()
        coffee = Coffee()

        print("\nMaking tea ...")
        tea.prepare_recipe()

        print("\nMaking coffee ...")
        coffee.prepare_recipe()

if __name__ == "__main__":
    BeverageTest.main()

File: hw/vizcaino/u3/hw26TemplateMethod/hw26TemplateMethodJavaScript/beverageTest.js
const Tea = require('./tea');
const Coffee = require('./coffee');

class BeverageTest {
    static async main() {
        const tea = new Tea();
        const coffee = new Coffee();

        console.log("\nMaking tea ...");
        await tea.prepareRecipe();

        console.log("\nMaking coffee ...");
        await coffee.prepareRecipe();
    }
}

BeverageTest.main();